.changeset/rare-books-own.md

```diff
@@ -1,13 +1,15 @@
 ---
-'babel-plugin-emotion': major
+'@emotion/babel-plugin': major
```

Comment:
this is somewhat a weird changeset, it shouldn't appear in generated changelogs, dont you think?

Reply:
Yeah, we can remove it before we do the actual release.